metadata/content.go

```diff
@@ -530,12 +530,14 @@ func writeInfo(info *content.Info, bkt *bolt.Bucket) error {
 	return bkt.Put(bucketKeySize, sizeEncoded)
 }
 
-func (cs *contentStore) garbageCollect(ctx context.Context) error {
-	lt1 := time.Now()
+func (cs *contentStore) garbageCollect(ctx context.Context) (d time.Duration, err error) {
```

Comment:
Should these gc times be registered in prometheus metrics?

Reply:
I think so, was going to explore that in a follow up. I would want to track the different pause times as well as overall GC times